federatedscope/core/workers/server.py

```
@@ -646,8 +646,11 @@ def broadcast_model_para(self,
                     timestamp=self.cur_timestamp,
                     content=model_para))
         if self._cfg.federate.online_aggr:
-            for idx in range(self.model_num):
-                self.aggregators[idx].reset()
+            try:
+                for idx in range(self.model_num):
+                    self.aggregators[idx].reset()
+            except:
```

we should not ignore exceptions happening here